langchain/src/llms/bedrock-chat.ts

```diff
@@ -0,0 +1,267 @@
+import { SignatureV4 } from "@aws-sdk/signature-v4";
+import { defaultProvider } from "@aws-sdk/credential-provider-node";
```

Comment:
Let's combine this into the original `llms/bedrock.ts` and call it adding streaming support!

Reply:
ok, sounds good.  I will work towards that now.  Thank you for the review